kratos/utilities/timer.cpp

```diff
@@ -0,0 +1,128 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//
+
+// System includes
+#include <ctime>
```

Comment:
I would replace this for std::chrono, then it is also independent of OMP
@msandre [did some tests](https://github.com/KratosMultiphysics/Kratos/pull/1478#issuecomment-366240165)